<launch>
  <include file="$(find multilayer_object_tracking)/launch/object_model_annotation.launch" />
  <!--<include file="$(find multilayer_object_tracking)/launch/supervoxel_segmentation.launch" />-->
  <node pkg="multilayer_object_tracking" type="multilayer_object_tracking_node"
	name="multilayer_object_tracking" output="screen" >
    <remap from="input_cloud" to="/camera/depth_registered/points" />
    <remap from="input_obj_cloud" to="/object_model/output/cloud" />
    <remap from="input_pose" to="/particle_filter_tracker/track_result_pose" />
    <remap from="input_obj_pose" to="/object_model/output/pose" />
  </node>
  <node pkg="rqt_reconfigure" type="rqt_reconfigure" name="rqt_reconfigure"/>
</launch>
